recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Make sure we are in the right place
+cd $SRC_DIR
+
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+
+# libcuda.so is a machine specific library. For building, we use
+# libcuda.so from stubs. For runtime, users must specify the location
+# of libcuda.so.1 in the environment variable LD_LIBRARY_PATH.
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib/stubs -Wl,-rpath-link,$PREFIX/lib/stubs"
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCMAKE_LIBRARY_PATH=$PREFIX/lib/stubs"
+
+# Enforce PREFIX instead of BUILD_PREFIX:
+export ZLIB_ROOT=$PREFIX
+export LibArchive_ROOT=$PREFIX
+export Curses_ROOT=$PREFIX
+export Glog_ROOT=$PREFIX
+export Snappy_ROOT=$PREFIX
+export Boost_ROOT=$PREFIX
+export PNG_ROOT=$PREFIX
+export GDAL_ROOT=$PREFIX
+export BLOSC_ROOT=$PREFIX
```

Comment:
Add  `-DCMAKE_PREFIX_PATH=$PREFIX` and you can remove all of these

Reply:
Nice. Thanks!

Reply:
Done